packages/nextjs/app/builders/[0xC4Aad525854Cc8d21122Bee8CcC1d9c3cEcBb859]/page.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+
+import { useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import type { NextPage } from "next";
+import { FaGithub, FaLinkedin, FaTwitter } from "react-icons/fa";
+import { Address } from "~~/components/scaffold-eth";
+
+const BuilderProfile: NextPage = () => {
```

Comment:
Please use a more unique component name, preferably something that includes your name or nickname.